### PR TITLE
Add timezone to ones missing

### PR DESCRIPTION
**Pull request by nicobav**
There was missing timezone declaration in various places using datetimes. This adds the tzinfo to those.

Some is still missing since getting warnings but cannot found the source yet so this is the changeset for now.

## Change log
- Timzones to various places

### CODE REVIEW

File: merchants/verkkokauppa/helpers.py [hunk 1]
```diff
@@ -23,7 +23,7 @@
 def parse_datetime(string: Optional[str]) -> Optional[datetime]:
     if string is None:
         return None
-    return datetime.strptime(string, "%Y-%m-%dT%H:%M:%S.%f")
+    return datetime.strptime(string, "%Y-%m-%dT%H:%M:%S.%f")  # what is the timezone?
```
Comment (nicobav) [inline merchants/verkkokauppa/helpers.py:26]: I pondered how does the verkkokauppa work with timezones. Does it send the times without zone info and we expect them to be UTC or UTC+3 🕵️

Comment (saaste) [reply merchants/verkkokauppa/helpers.py:26]: I asked this from Severi and he told me all times are UTC

**Review by saaste — APPROVED** [review]
Cooool beans!

File: merchants/verkkokauppa/helpers.py (hunk 1 above)
Comment (saaste) [reply merchants/verkkokauppa/helpers.py:26]: Oh and feel free to change to to explicitly read the datetime as UTC. I already approved the rest of the PR 😄

**Pull request merged**